dali/operators/reader/video_reader_decoder_op_test.cc

```diff
@@ -168,6 +172,15 @@ void VideoReaderDecoderBaseTest::RunShuffleTest<dali::CPUBackend>() {
     RunShuffleTestImpl<dali::CPUBackend>("cpu", dali::CPU_ONLY_DEVICE_ID);
 }
 
+template<>
+int VideoReaderDecoderBaseTest::GetFrameNo(
```

Comment:
```suggestion
int VideoReaderDecoderBaseTest::GetFrameIdx(
```
...if going with the `idx` suggestion.